docs/content/usage/scenarios/custom-scenario/network-breach.md

```diff
@@ -18,7 +18,7 @@ Infection Monkey will help you assess the impact of a future breach by attemptin
 - **Exploits -> Exploits** Here you can review the exploits the Infection Monkey will be using. By default all
 safe exploiters are selected.
 - **Exploits -> Credentials** This configuration value will be used for brute-forcing. The Infection Monkey uses the most popular default passwords and usernames, but feel free to adjust it according to the default passwords common in your network. Keep in mind a longer list means longer scanning times.
-- **Network -> Scope** Make sure to properly configure the scope of the scan. You can select **Local network scan**
+- **Network -> Scope** Make sure to properly configure the scope of the scan. You can select **Scan Agent's networks**
```

Comment:
I think the paths are changed:

Propagation -> Network analysis -> Network 

?